Packs/GoogleChronicleBackstory/Integrations/GoogleChronicleBackstory/GoogleChronicleBackstory.yml

```diff
@@ -1,6 +1,6 @@
 category: Analytics & SIEM
 commonfields:
-  id: Google Chronicle Backstory
+  id: Chronicle
```

Comment:
It is not allowed to change the integration id. It will break backward compatibility for the customers . You are allowed only to change `display` field

Reply:
Okay, you put it back, right?